Source/qol/stash.h

```diff
@@ -68,30 +110,111 @@ constexpr Point InvalidStashPoint { -1, -1 };
+ * @return x,y screen space coordinates of the provided slot.
+ */
 Point GetStashSlotCoord(Point slot);
+
+/**
+ * @brief Initialize stash, setting gold to 0 and loading panel/button art.
+ */
 void InitStash();
+
+/**
+ * @brief Release stash art assets.
+ */
 void FreeStashGFX();
-void TransferItemToInventory(Player &player, uint16_t itemId);
+
+/**
+ * @brief Transfer item by itemIndex (itemId-1) to player, updating stash grid as well.
+ * @param player reference to active player.
+ * @param itemIndex 0-based index of the item as it exists in the stashList.
+ */
+void TransferItemToInventory(Player &player, uint16_t itemIndex);
+
 /**
  * @brief Render the inventory panel to the given buffer.
  */
 void DrawStash(const Surface &out);
+
+/**
+ * @brief Responds to action against a screen position within the stash panel (ex: mouse click).
+ * @param mousePosition x,y screen coordinates of mouse during action.
+ * @param isShiftHeld bool for whether shift key is down (auto transfer to belt).
+ * @param isCtrlHeld bool for whether ctrl key is down (auto transfer to inventory).
+ */
 void CheckStashItem(Point mousePosition, bool isShiftHeld = false, bool isCtrlHeld = false);
-bool UseStashItem(uint16_t cii);
+
+/**
+ * @brief Attempt to use item from stash.
+ * @param itemIndex 0-based index of the item as it exists in the stashList.
+ * @return 'True' if item used, 'False' if failure to use item.
+ */
+bool UseStashItem(uint16_t itemIndex);
+
+/**
+ * @brief Handle stash highlight and stats pop-up, if an item is under the cursor.
+ * @param mousePosition x,y screen coordinates of mouse.
+ * @return 0-based index of the item as it exists in the stashList, or InvalidItemId if nothing is under the cursor.
+ */
 uint16_t CheckStashHLight(Point mousePosition);
+
+/**
+ * @brief Perform UI action based on x,y screen position of mouse, responding to StashButtonPressed value.
+ * @param mousePosition x,y screen coordinates of mouse at release.
+ */
 void CheckStashButtonRelease(Point mousePosition);
+
+/**
+ * @brief Store x,y screen position of mouse, setting StashButtonPressed for use in CheckStashButtonRelease.
+ * @param mousePosition x,y screen coordinates of mouse at press.
+ */
 void CheckStashButtonPress(Point mousePosition);
 
+/**
+ * @brief Begin the gold withdraw process, initializing the withdraw dialog popup.
+ */
 void StartGoldWithdraw();
+
+/**
+ * @brief Respond to key press while the gold withdraw process is in progress.
+ * @param vkey SDL keycode of the pressed key.
+ */
 void WithdrawGoldKeyPress(SDL_Keycode vkey);
+
+/**
+ * @brief Rendering the gold withdraw popup and the current value.
+ * @param out SDL surface upon which to draw the output.
+ * @param amount entered withdraw amount to populate in the withdraw popup.
+ */
 void DrawGoldWithdraw(const Surface &out, int amount);
+
+/**
+ * @brief Closes the gold withdraw popup and stop capturing text input.

```

Comment:
```suggestion
 * @brief Close the gold withdraw popup and stop capturing text input.
```
Let's be consistent, closes/stops or close/stop